### PR TITLE
New() -> Initialize()

### PR DESCRIPTION
**Pull request by nicbn**
Now without runtimes I think

### CODE REVIEW

File: code/modules/awaymissions/exile.dm [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	name = "exile implants"
 	req_access = list(ACCESS_HOS)
 
-/obj/structure/closet/secure_closet/exile/New()
-	..()
+/obj/structure/closet/secure_closet/exile/Initialize()
+	. = ..()
 	new /obj/item/implanter/exile(src)
```
Comment (SpaceManiac) [inline code/modules/awaymissions/exile.dm]: This should probably be moved to `PopulateContents`.

File: code/modules/detectivework/evidence.dm [hunk 2]
```diff
@@ -84,12 +84,11 @@
 	name = "evidence bag box"
 	desc = "A box claiming to contain evidence bags."
 
-/obj/item/storage/box/evidence/New()
+/obj/item/storage/box/evidence/Initialize()
+	. = ..()
 	new /obj/item/evidencebag(src)
```
Comment (SpaceManiac) [inline code/modules/detectivework/evidence.dm]: This should probably be moved to `PopulateContents`.

File: code/game/objects/items/religion.dm [hunk 3]
```diff
@@ -347,20 +347,18 @@
 	desc = "This armour is said to be based on the armor of kings on another world thousands of years ago, who tended to assassinate, conspire, and plot against everyone who tried to do the same to them.  Some things never change."
 
 
-/obj/item/storage/box/itemset/crusader/blue/New()
-	..()
+/obj/item/storage/box/itemset/crusader/blue/Initialize()
+	. = ..()
 	contents = list()
-	sleep(1)
 	new /obj/item/clothing/suit/armor/plate/crusader/blue(src)
```
Comment (SpaceManiac) [inline code/game/objects/items/religion.dm]: This probably belongs in `PopulateContents`.

File: code/game/objects/items/religion.dm [hunk 4]
```diff
@@ -347,20 +347,18 @@
...
 	new /obj/item/clothing/shoes/plate/blue(src)
 
 
-/obj/item/storage/box/itemset/crusader/red/New()
-	..()
+/obj/item/storage/box/itemset/crusader/red/Initialize()
+	. = ..()
 	contents = list()
-	sleep(1)
 	new /obj/item/clothing/suit/armor/plate/crusader/red(src)
```
Comment (SpaceManiac) [inline code/game/objects/items/religion.dm]: This probably belongs in `PopulateContents`.

File: code/game/objects/effects/landmarks.dm [hunk 5]
```diff
@@ -48,6 +48,9 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 			GLOB.jobspawn_overrides[name] = list()
 		GLOB.jobspawn_overrides[name] += src
 	..()
+
+/obj/effect/landmark/start/Initialize()
+	. = ..()
```
Comment (SpaceManiac) [inline code/game/objects/effects/landmarks.dm]: Not sure about this, if the other stuff is needed right away than the tag might be as well

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
Thanks for the code improvements. There's a couple of things which used to be in New, are now in Initialize, but really belong in PopulateContents that it would be nice to handle, or we can put that off to a future PR.

**Comment by optimumtact** [comment]
@nicbn hello?

**Comment by nicbn** [comment]
I'm not dead, one moment

File: code/game/objects/items/religion.dm [hunk 6]
```diff
@@ -347,20 +347,18 @@
 	desc = "This armour is said to be based on the armor of kings on another world thousands of years ago, who tended to assassinate, conspire, and plot against everyone who tried to do the same to them.  Some things never change."
 
 
-/obj/item/storage/box/itemset/crusader/blue/New()
-	..()
+/obj/item/storage/box/itemset/crusader/blue/PopulateContents()
+	. = ..()
 	contents = list()
```
Comment (SpaceManiac) [inline code/game/objects/items/religion.dm]: These two lines (parent call and contents clear) cancel each other out and can both be removed.
It's considered normal for `PopulateContents` not to call parent.
Ditto below.

File: code/game/objects/structures/tables_racks.dm [hunk 7]
```diff
@@ -298,25 +298,16 @@
...
 
-/obj/structure/table/wood/fancy/New()
-	// New() is used so that the /black subtype can override `icon` easily and
-	// the correct value will be used by the smoothing subsystem.
+/obj/structure/table/wood/fancy/Initialize()
 	. = ..()
-	// Needs to be set dynamically because table smooth sprites are 32x34,
-	// which the editor treats as a two-tile-tall object. The sprites are that
-	// size so that the north/south corners look nice - examine the detail on
-	// the sprites in the editor to see why.
```
Comment (SpaceManiac) [inline code/game/objects/structures/tables_racks.dm]: I would suggest keeping this comment. Also, is this tested? I have a feeling it doesn't work.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/tables_racks.dm (hunk 7 above)
Comment (nicbn) [reply code/game/objects/structures/tables_racks.dm]: I'll add a comment. I tested it some time ago and it should work (can't test right now, trying to get wine to work)

**Comment by nicbn** [comment]
@SpaceManiac

**Review by SpaceManiac — APPROVED** [review]
fingers crossed

**Pull request merged**